marshal/base-types.c

```diff
@@ -128,9 +134,25 @@ type##_Unmarshal ( \
          (uintptr_t)buffer, \
          (uintptr_t)dest, \
          local_offset); \
-    type tmp = 0; \
+\
     memcpy (&tmp, &buffer [local_offset], sizeof (tmp)); \
```

Comment:
Agree. I left the spaces as they were. We can update the style changes in separate commits after we agree on what we want.